### PR TITLE
Fixed some warnings

### PR DESCRIPTION
**Pull request by killerfrienddk**

### CODE REVIEW

**Review by MrCakeSlayer — APPROVED** [review]

File: src/Discord.Net.Core/Entities/Users/IGuildUser.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public interface IGuildUser : IUser, IVoiceState
         /// </summary>
         /// <example>
         ///     <para>The following example checks if the current user has the ability to send a message with attachment in
-        ///     this channel; if so, uploads a file via <see cref="IMessageChannel.SendFileAsync(string, string, bool, Embed, RequestOptions, bool, AllowedMentions, MessageReference)"/>.</para>
+        ///     this channel; if so, uploads a file via IMessageChannel.SSendFileAsync(string filePath, string text = null, bool isTTS = false, Embed embed = null, RequestOptions options = null, bool isSpoiler = false, AllowedMentions allowedMentions = null, MessageReference messageReference = null, MessageComponent component = null, ISticker[] stickers = null).</para>
```
Comment (quinchs) [inline src/Discord.Net.Core/Entities/Users/IGuildUser.cs]: why was the see tag removed?

File: src/Discord.Net.WebSocket/Entities/Interaction/SocketInteraction.cs [hunk 2]
```diff
@@ -118,7 +118,7 @@ internal virtual void Update(Model model)
         /// <summary>
         ///     Responds to an Interaction with type <see cref="InteractionResponseType.ChannelMessageWithSource"/>.
         /// <para>
-        ///     If you have <see cref="DiscordSocketConfig.AlwaysAcknowledgeInteractions"/> set to <see langword="true"/>, You should use
+        ///     If you have DiscordSocketConfig.AlwaysAcknowledgeInteractions set to <see langword="true"/>, You should use
```
Comment (quinchs) [inline src/Discord.Net.WebSocket/Entities/Interaction/SocketInteraction.cs]: `AlwaysAcknowledgeInteractions ` is deprecated, para should be removed

**Review by quinchs — CHANGES_REQUESTED** [review]

File: src/Discord.Net.Core/Entities/Users/IGuildUser.cs [hunk 3]
```diff
@@ -83,7 +83,7 @@ public interface IGuildUser : IUser, IVoiceState
         /// </summary>
         /// <example>
         ///     <para>The following example checks if the current user has the ability to send a message with attachment in
-        ///     this channel; if so, uploads a file via <see cref="IMessageChannel.SendFileAsync(string, string, bool, Embed, RequestOptions, bool, AllowedMentions, MessageReference)"/>.</para>
+        ///     this channel; if so, uploads a file via IMessageChannel.SendFileAsync(string filePath, string text = null, bool isTTS = false, Embed embed = null, RequestOptions options = null, bool isSpoiler = false, AllowedMentions allowedMentions = null, MessageReference messageReference = null, MessageComponent component = null, ISticker[] stickers = null).</para>
```
Comment (quinchs) [inline src/Discord.Net.Core/Entities/Users/IGuildUser.cs]: no see tag

File: src/Discord.Net.Core/Extensions/MessageExtensions.cs [hunk 4]
```diff
@@ -59,6 +59,7 @@ public static async Task AddReactionsAsync(this IUserMessage msg, IEmote[] react
         /// </code>
         /// </example>
         /// <param name="msg">The message to remove reactions from.</param>
+        /// <param name="user">The user who removed the reaction.</param>
         /// <param name="reactions">An array of reactions to remove from the message</param>
```
Comment (quinchs) [inline src/Discord.Net.Core/Extensions/MessageExtensions.cs]: Grammar, add `.` to the end

File: src/Discord.Net.Core/Extensions/MessageExtensions.cs [hunk 5]
```diff
@@ -84,6 +86,8 @@ public static async Task RemoveReactionsAsync(this IUserMessage msg, IUser user,
         ///     If <c>null</c>, all mentioned roles and users will be notified.
         /// </param>
         /// <param name="options">The options to be used when sending the request.</param>
+        /// <param name="components">The message components to be included with this message. Used for interactions</param>
```
Comment (quinchs) [inline src/Discord.Net.Core/Extensions/MessageExtensions.cs]: Grammar, no `.`

File: src/Discord.Net.Core/Extensions/UserExtensions.cs [hunk 6]
```diff
@@ -85,6 +85,7 @@ public static class UserExtensions
         /// <param name="isTTS">Whether the message should be read aloud by Discord or not.</param>
         /// <param name="embed">The <see cref="EmbedType.Rich"/> <see cref="Embed"/> to be sent.</param>
         /// <param name="options">The options to be used when sending the request.</param>
+        /// <param name="component">The message component to be included with this message. Used for interactions</param>
```
Comment (quinchs) [inline src/Discord.Net.Core/Extensions/UserExtensions.cs]: Grammar, no `.`

File: src/Discord.Net.Core/Extensions/UserExtensions.cs [hunk 7]
```diff
@@ -142,6 +143,7 @@ public static class UserExtensions
...
```
Comment (quinchs) [inline src/Discord.Net.Core/Extensions/UserExtensions.cs]: Grammar, no `.`

**Review by quinchs — CHANGES_REQUESTED** [review]

**Review by quinchs — APPROVED** [review]

**Pull request merged**